concent_api/core/models.py

```diff
@@ -379,6 +391,7 @@ class Meta:
 
     # Related messages
     task_to_compute = OneToOneField(StoredMessage, related_name='subtasks_for_task_to_compute')
+    want_to_compute_task = OneToOneField(StoredMessage, related_name='subtasks_for_want_to_compute_task')
     report_computed_task = OneToOneField(StoredMessage, related_name='subtasks_for_report_computed_task')
     ack_report_computed_task = OneToOneField(StoredMessage, blank=True, null=True, related_name='subtasks_for_ack_report_computed_task')
     reject_report_computed_task = OneToOneField(StoredMessage, blank=True, null=True, related_name='subtasks_for_reject_report_computed_task')
```

Comment:
I can't find changes to validation that `want_to_compute` stored separately is the same message as `want_to_compute` nested inside another messages. Please add this validation in the same way as it is now for `TTC` and `RCT`.

Reply:
As we spoke, validation of `want_to_compute` is redundant, because it's being done when we validate `task_to_compute`. I've try to implement it, and then I notice it can't be tested, because `ValidationError` is raised sooner.